frontend/src/components/pages/GlobalForms/WaiversTab.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiverSectionCard from "./WaiverSectionCard";
+  
+  interface WaiversTabProps {
+    clauses: Array<ClauseInfo>;
+  }
+  const WaiversTab = ({
+    clauses
+  }: WaiversTabProps): JSX.Element => {
+
+    function intToChar(number: number) {
+        const code = 'A'.charCodeAt(0);  
+        return String.fromCharCode(code + number);
+    }
+    return (
+        <>
+        {clauses.length && (clauses).map((clause, idx)=> {
```

Comment:
You should be able to map over an empty array. So you should be able to replace this with:
```
{
    clauses.map((clause, index) => {..})
}
```